virttest/libvirt_vm.py

```diff
@@ -937,10 +937,10 @@ def add_nic(help_text, nic_params):
                 if (
                     nettype
                     and nic_queues
-                    and has_sub_option("network", "driver_queues")
+                    and has_sub_option("network", r"driver[._]queues")
                 ):
                     result += ",driver_queues=%s" % nic_queues
-                    if nic_driver and has_sub_option("network", "driver_name"):
+                    if nic_driver and has_sub_option("network", r"driver[._]name"):
```

Comment:
Same as the above. Thanks.

Reply:
Fixed. please help review them again, thanks a lot.